requirements-unlocked.txt

```diff
@@ -0,0 +1,2 @@
+pandas
```

Comment:
@matentzn I'm not opposed to `poetry` or `tox`, but this native `pip` way of doing things is the fastest / easiest way for me to do this right now.

I hope it's OK to add these dependencies, even though I have only one script that would use them right now.